3p/iframe-messaging-client.js

```diff
@@ -101,6 +101,15 @@ export class IframeMessagingClient {
    */
   setupEventListener_() {
     listen(this.win_, 'message', event => {
+      if (this.hostWindowIsActuallyAnIframe_()) {
+        // this.hostWindow_ can now be set to an iframe, after it has been
```

Comment:
oh, wait, you wanted to make the iframe client a "host". which should not be a case after you switched to SubscriptionApi.